release-gap-package

```diff
@@ -181,6 +190,14 @@ if [ ! -d "$WEB_DIR" ] ; then
     error "could not find 'webdir' with clone of your gh-pages branch"
 fi
 
+# Check for presence of update.g file
+if [ "x$UPDATE_FILE" = x ] ; then
+    UPDATE_FILE="$WEB_DIR/update.g"
+fi
+if [ ! -f "$UPDATE_FILE" ] ; then
+    error "could not find update.g file"
```

Comment:
Maybe this would be more helpful?
```suggestion
    error "could not find ${UPDATE_FILE} file"
```

Reply:
Good idea, I have pushed a slightly different suggestion (a841bcfb444b7d8687ddf6793e9390ba65a53440) which is also helpful in case one passes the empty string to --update-file.